apps/dashboard/app/views/shared/_insufficient_balance_resource.html.erb

```diff
@@ -1,10 +1,8 @@
-<div>
-  <p class="fs-4 mb-1 alert-heading">
-    <%= t('dashboard.balance_warning_prefix_html', units_balance: balance.units_balance, unit: balance.unit) %>
-    <strong><%= balance.project_type %> <%= balance.balance_object %></strong>
-  </p>
-  <p><%= balance.to_s %>. <%= t('dashboard.balance_additional_message', balanace_units: balance.balanace_units, unit: balance.unit) %></p>
-</div>
-<p class="text-muted d-none d-md-block p-2">
+<h4 class="alert-heading">
```

Comment:
Actual `h4` tags I don't think make sense for accessibility reasons. That is, for a h4 to be on the page and make sense for a screen reader user, it implies that there's also h1 - h3 tags above it. So a screen reader user would see this as very out of place.

For sizing you can use the class `h4` on a div if you're just trying to get it to be large.